bin/disco_ssh.py

```diff
@@ -133,16 +135,31 @@ def build_ssh_cmd(self, ips):
         """
         Given a list of ip addresses, build an ssh command to tunnel through n-1 ips to reach the n-th ip
         """
-        command = " ".join([
-            "ssh -At {} {}".format(SSH_OPTIONS, ip)
-            for ip in ips])
+        if self.tunnel:
+            lport, rport = self.tunnel.split(":")
+            host = self.args["<host>"]
```

Comment:
So, normally the "host" argument is _not_ addressable, because we're doing a host search for relevant EC2 instances in the given environment.  So we're using the same argument for different things, which is a little weird.